cron/dump_clean_daily

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Performs a filtered dump of the measurements table.
+#
+# The filtering is the same as that used by the iOS app.
+#
+# This differs from dump_measurements, which does not filter data at all.
+# The schema returned is also different; redundant/unused columns have been eliminated, and id/user_id finally included.
+# The ORDER BY clause was removed for performance.
+
+set -euo pipefail
+
+source cron_env.sh
+
+cd ../public/system
+
+psql -f "${CRON_DIR}/dump_clean_daily.sql"
+
+tar -czf dump_clean_daily.tar.gz dump_clean_daily-out.csv
+mv dump_clean_daily-out.csv dump_clean_daily.csv
```

Comment:
I'm not sure what the purpose of this line is.

Reply:
I have no idea as well, might be some internal use? I follow this [pattern](https://github.com/Safecast/safecastapi/blob/master/cron/dump_clean#L20)